torchft/lighthouse_test.py

```diff
@@ -1,10 +1,11 @@
 import time
+from datetime import timedelta
 from unittest import TestCase
 
 import torch.distributed as dist
 
 from torchft import Manager, ProcessGroupGloo
-from torchft._torchft import LighthouseServer
+from torchft._torchft import LighthouseClient, LighthouseServer
```

Comment:
We should expose the `LighthouseClient` in `torchft.coordination`. That way it'll get documented on the site

Reply:
sure.